codesign/test/file_codesing_visitor_test.dart

```diff
@@ -37,6 +37,7 @@ void main() {
         tempDir: tempDir,
```

Comment:
please rename this file `file_codesign_visitor_test.dart`

Reply:
This is embarrassing. Very true, renamed.